README.md

```diff
@@ -1,209 +1,113 @@
-
-Afterwards, calling the creation of Client with default parameters will load the configurations
-previously saved:
-
-```python
-from darwin.client import Client
-
-client = Client.local()
-```
-
-
-#### Listing local and remote datasets
-
-Print a list of local existing projects
-
-```python
-from darwin.client import Client
-
-client = Client.local()
-for p in client.list_local_datasets():
-    print(p.name)
-```
-
-Print a list of remote projects accessible by the current user.
-Note: the list will include only those datasets which belong in the team where the user is currently 
-authenticated. 
+## Installation
 
-```python
-from darwin.client import Client
+### Standard
 
-client = Client.local()
-for dataset in client.list_remote_datasets():
-    print(dataset.slug, dataset.image_count)
 ```
-
-
-#### Create/remove a dataset 
-
-Dataset creation is handled by the client:
-
-```python
-from darwin.client import Client
-
-client = Client.local()
-client.create_dataset(name="This Is My New Dataset")
+pip install darwin-py
 ```
+You can now type `darwin` in your terminal and access the command line interface.
 
-Whereas dataset removal is handled directly by the dataset itself:
+### Development
 
-```python
-from darwin.client import Client
+After cloning the repository:
 
-client = Client.local()
-dataset = client.get_remote_dataset(slug="this-is-my-new-dataset")
-dataset.remove_remote()
 ```
-
-
-#### Upload/download data to/from a remote dataset
-
-To upload data to an existing remote project there are several solutions.
-A simple one is to update the local folder where the dataset is located and then upload the
-files to the remote dataset. 
-
-```python
-from darwin.client import Client
-
-client = Client.local()
-dataset = client.get_remote_dataset(slug="example-dataset")
-progress = dataset.push()
+pip install --editable .
 ```
 
-Note that `dataset.push()` takes an optional parameter `source_folder` with which is possible
-to specify another location from which fetch the images to upload.
-To download a remote project, images and annotations, in the projects directory 
-(specified in the authentication process [default: ~/.darwin/projects]).
 
-```python
-from darwin.client import Client
 
-client = Client.local()
-dataset = client.get_remote_dataset(slug="example-dataset")
-dataset.pull()
-```
+---
 
+## Usage as a Command Line Interface (CLI)
 
----
+Once installed, `darwin` is accessible as a command line tool.
+A useful way to navigate the CLI usage is trough the help command `-h/--help` which will 
+provide additional information for each command available. 
 
-### Command line
 
-`darwin` is also accessible as a command line tool.
+### Client Authentication 
 
+To access perform remote operation on Darwin you first need to authenticate.
+This requires a [team-specific API-key](https://darwin.v7labs.com/?settings=api-keys).  
+If you do not already have a Darwin account, you can [register for free](https://darwin.v7labs.com).
```

Comment:
You can't anymore register for free, let's get rid of this line, or replace it by something like
```suggestion
If you do not already have a Darwin account, you can [contact us](https://darwin.v7labs.com/whatever-link) and we can set one up for you.
```